cni/network/network_windows.go

```diff
@@ -18,6 +18,10 @@ import (
 	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
 )
 
+const (
+	snatOnHostConfigFileName = "C:\\Windows\\Temp\\disableSnatOnHost"
```

Comment:
did we verify this is getting removed after reboot?

Reply:
works win10 but not on server 2019...

Reply:
then this is problem right? can we remove the file on reboot?

Reply:
yeah I need to add that, that's why I was asking about the registry key to automatically move/delete file